dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -111,13 +112,25 @@ def self.application_status(user)
       has_confirmed_school: application.try(:has_confirmed_school?) || false,
       school_id: school_id,
       school_name: school_id ? School.find(school_id).name : nil,
+      school_high_needs_eligible: School.find(school_id).try(:maker_high_needs?),
```

Comment:
I liked your proposal to move the `maker_high_needs?` implementation out of the `School` model and into this one.

Reply:
Alternatively, we could create a [policy object](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/policies/README.md), a clean-architecture Ruby pattern @Hamms introduced into our project this year.

Reply:
I left this as-is for now -- could fix after break if we want.